azure-pipelines.yml

```diff
@@ -40,22 +40,14 @@ jobs:
 - job: Docker
   strategy:
     matrix: 
-      gcc7:
-        docker_target: gcc7
-      gcc8:
-        docker_target: gcc8
-      gcc9:
-        docker_target: gcc9
-      gcc11:
-        docker_target: gcc11
-      clang10:
-        docker_target: clang10
-      clang11:
-        docker_target: clang11
-      clang12:
-        docker_target: clang12
-      sycl:
-        docker_target: sycl
+      gcc:
+        docker_target: gcc
+      clang:
+        docker_target: clang
```

Comment:
What about an intel build?

Reply:
Coming soon, need to fix the Intel image which currently fails to build :'(